generator/sbpg/targets/java.py

```diff
@@ -67,7 +67,7 @@ def commentify(environment: Environment,
   """
   Builds a comment.
   """
-  return indented_wordwrap(environment, value, indent=(" " * indent) + " * ", first=False)
+  return indented_wordwrap(environment, value.replace("<", "{@code <}").replace(">", "{@code >}"), indent=(" " * indent) + " * ", first=False)
```

Comment:
Wasn't `@literal` the one you originally proposed to use? It seems to me to be more appropriate here than `code`?